### PR TITLE
Faster computer vision on bebop using isp

### PR DESCRIPTION
**Pull request by dewagter**
This is the code in use in all the bebop projects where fast good front camera images are important: 

 - drone race
 - course
 - swarming
 - [ ] add an option for the original resolution
 - [ ] merge the VGA resolution option

### CODE REVIEW

File: conf/Makefile.arm-linux-toolchain [hunk 1]
```diff
@@ -26,6 +26,8 @@ ifndef HARD_FLOAT
 endif
 endif
 
+PREFIX=/opt/arm-2012.03/bin/arm-none-linux-gnueabi
```
Comment (flixr) [inline conf/Makefile.arm-linux-toolchain]: Will break other linux builds

Comment (dewagter) [reply conf/Makefile.arm-linux-toolchain]: fixed... will be addressed separately: see https://github.com/paparazzi/paparazzi/issues/2169

**Comment by OpenUAS** [comment]
Any open objections or can we merge it? Then I can start retest on BB,BB2 and prelim Disco

**Comment by kirkscheper** [comment]
@OpenUAS This is still marked as incomplete by Christohpe. Also see the check boxes in the original comment.

**Comment by kirkscheper** [comment]
This has been implemented with #2364

**Pull request closed without merging**